versions.yaml

```diff
@@ -31,6 +31,7 @@ module-sets:
     modules:
       - go.opentelemetry.io/otel/log
       - go.opentelemetry.io/otel/sdk/log
+      - go.opentelemetry.io/otel/sdk/log/logtest
```

Comment:
@MrAlias, we should not release this module as well, right?
Notice, that this module is used in other modules, but it should not any problem given it is used only in tests (because of similar reasons to https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6763)

Reference: https://github.com/open-telemetry/opentelemetry-go/pull/6465#discussion_r2004336953

Reply:
> @MrAlias, we should not release this module as well, right?

SGTM

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/6466/commits/42eb465f2bd18329a8039f3cd7dec1ba5855aad2